src/scene/visitors/elementrgbbitmapsetter.cpp

```diff
@@ -9,6 +9,11 @@
 #include "bitmaps.h"
 
 
+inline uint16_t crgbToRGB565(CRGB color) {
```

Comment:
Color conversion functions are already defined in `include/colorconversion.h`, no need to define one here. This exact one is on line 46.